configurations/performance/rust-predefined-throughput-steps-cql-stress.yaml

```diff
@@ -1,3 +1,6 @@
+# Define the commands to be used in the write-load test. The purpose of this command is to prepare the schema.
+prepare_stress_cmd: "cql-stress-cassandra-stress write cl=QUORUM n=1 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate 'threads=1' -col 'size=FIXED(1024) n=FIXED(1)'"
```

Comment:
why one is QUORUM and the other is ALL ?

Reply:
fixed